### DISCUSSION TITLE
Add toggle button inside Painting Editor to switch mode

### DISCUSSION THREAD

**Discussion by thanglq1** [hm21/pro_image_editor#78] [Q&A] [answered]
Add a toggle button inside the Painting Editor to easily switch between normal crop and round crop modes

**Comment by hm21** [comment, marked as answer]
Do you mean inside the Crop-Rotate-Editor? Or you required to crop the image inside the paint-editor?

If you want to change in the Crop-Rotate-Editor, you can edit the app or bottom bar like in the [example](https://github.com/hm21/pro_image_editor/blob/stable/example/lib/pages/custom_appbar_bottombar_example.dart). But be careful the round cropper only supports the aspect ratio 1×1 so if you allow the user to change the aspect ratio in the round cropper can it make some issues. 

If you would like to change in the paint editor, it is not possible because the paint editor is not built for this case. What you can do is to create your own extended [paint-editor](https://github.com/hm21/pro_image_editor/blob/stable/lib/modules/paint_editor/paint_editor.dart) if you copy the paint editor code and add a method where you can edit the `transformConfigs`. This will require a lot of logic, but you can copy/paste from the code I use in the [crop-rotate-editor](https://github.com/hm21/pro_image_editor/blob/stable/lib/modules/crop_rotate_editor/crop_rotate_editor.dart) and just make the necessary changes.

**Comment by thanglq1** [comment]
Yes, I mean to add a toggle button inside the Crop-Rotate-Editor. Thank you for your suggestion. I will customize the bottom bar.
